core/src/cluster_slots_service/cluster_slots.rs

```diff
@@ -95,6 +95,35 @@ struct RowContent {
 }
 
 impl ClusterSlots {
+    #[inline]
```

Comment:
nit: Let's leave `inline` off of these 3 new functions and let the compiler do what it wants; these are constructors and only called once at startup so not like these are getting called in a hot path or anything